src/main/java/nextstep/subway/line/domain/Distance.java

```diff
@@ -15,6 +17,10 @@ public Distance(int distance) {
         this.distance = distance;
     }
 
+    public int getDistance() {
+        return distance;
+    }
+
     public Distance getAddedDistance(Distance mergingDistance) {
```

Comment:
```suggestion
    public Distance add(Distance distance) {
```
이렇게 써도 의미가 통하고 더 깔끔하지 않을까요?
getter가 아닌데 get을 붙이면 어떤 기능인지 헷갈립니다.


---

클린코드
p32
메서드 이름
... 접근자(Accessor), 변경자(Mutator), 조건자(Predicate)는 javabean 표준에 따라 값 앞에 get, set, is를 붙인다.

Reply:
get을 습관적으로 붙였던 것 같아요.
좀 더 명확한 의미의 이름을 부여해보겠습니다.